src/Components/Links.js

```diff
@@ -25,13 +32,13 @@ function Links({ links }) {
           .map((link, index) => (
             <Button
               className="p-p-3 p-m-2 p-button-outlined"
-              style={{ color: colors[link.icon] }}
+              style={{ color: 'white', backgroundColor: bgcolors[link.icon], borderRadius: '50px' }}
               key={`link.url_${index}`}
               onClick={() => goToLinkHandle(link.url)}
               role="link"
             >
-              <i className={`pi pi-${link.icon} p-px-2`}></i>
-              <span className="p-px-3">{link.name}</span>
+              <i className={`pi pi-${link.icon} p-px-2` } style= {{ fontSize: '30px' }} ></i>
```

Comment:
`px` will not allow it to resize`, also probably best to more to css file

Reply:
Which Pixel do I need to use then ?

Reply:
probably `em`